app/js/controllers/controller-outfit.js

```diff
@@ -404,6 +411,48 @@ angular.module('app').controller('OutfitController', ['$window', '$rootScope', '
       }
     }
     $scope.retrofitTotal = total;
+    updateAmmoCosts();
+  }
+
+  function updateAmmoCosts() {
+    var costs = $scope.ammoList = [];
+    var total = 0, i, l, item;
+
+    for (var g in { common: 1, internal: 1, hardpoints: 1 }) {
```

Comment:
common is not needed here since nothing in common/standard has ammo

`{ internal: 1, hardpoints: 1 }` should suffice